content/blog/2021/05/multicast-failure-routing/index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Routing multicast output after encountering partial failures"
+date: 2021-05-10
+draft: false
+authors: [Reji Mathews, Hokutor]
```

Comment:
These need to be the GitHub usernames, we use them to pull avatars from the GitHub profile, so you might want to set an image in your GitHub profile.

Reply:
Resolved . Falling back to github user id

Reply:
@zregvart - since its an array, could we add a co-author github ID too ?

Reply:
Yeah, you can add as many as you need here, but they need to be GitHub usernames.

Reply:
Thanks. and appreciate your quick response :)